pages/mesosphere/dcos/services/elastic/3.2.0-7.9.3/configuration/index.md

```diff
@@ -0,0 +1,389 @@
+---
+layout: layout.pug
+navigationTitle: Configuration
+excerpt: Configuration options for Elastic
+title: Configuration
+menuWeight: 20
+model: /mesosphere/dcos/services/elastic/data.yml
+render: mustache
+---
+
+#include /mesosphere/dcos/services/include/configuration-install-with-options.tmpl
+#include /mesosphere/dcos/services/include/configuration-service-settings.tmpl
+#include /mesosphere/dcos/services/include/configuration-regions.tmpl
+
+## Configuration Guidelines
+
+- Service name: This should be unique for each instance of the service that is running. It is also used as your cluster name.
+- Service user: This should be a non-root user who already exists on each agent. The default user is `nobody`.
+- X-Pack is installed by default and comes with a 30-day trial license.
+- Health check credentials: If you have X-Pack Security enabled, the health check will use the credentials specified in the configuration for authorization. We recommend that you create a specific {{ model.techName }} user/password for this with minimal capabilities rather than using the default superuser `{{ model.serviceName }}`.
```

Comment:
```suggestion
- Health check credentials: If you have X-Pack Security enabled, the health check will use the credentials specified in the configuration for authorization. We recommend that you create a specific `{{ model.techName }}` user/password for this with minimal capabilities rather than using the default superuser `{{ model.serviceName }}`.
```

Reply:
So, using these backticks here to make {{ model.techName }} to be in `code` for markdown or whatever, but it is without these throughout the doc, so maybe not use this one? Seems like the style guide for this is to not have it in backticks (`). So, ignore? Or not and change the rest? ¯\_(ツ)_/¯